src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,61 @@
+.calendar {
+  position: relative;
+  top: calc(50vh - #{$calendar-height} / 2);
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+
+  width: $calendar-width;
+  padding: $calendar-paddings;
+
+  background-color: $calendar-color;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    box-sizing: border-box;
+    width: $cell-size;
+    height: $cell-size;
+    border: $cell-border solid #000;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    background-color: $day-color;
+
+    transition: transform 0.5s;
```

Comment:
It would be great to create a variable for transition duration, so you can change it later if needed